R/decisiontreeutils.R

```diff
@@ -84,39 +73,38 @@ createDTParams <- function(config, data) {
 #' @param f_string string of function
 #' @param params list of decision tree params
 #' @return list with named parameters for f_string
-convertParamsToArgs <- function(f_string, params) {
-  if (f_string == "rpart") {
-    list(
-      data = params$data,
-      formula = params$formula,
-      weights = params$weights,
-      method = params$method,
-      parms = params$parms,
-      usesurrogate = params$usesurrogate,
-      minsplit <- params$minsplit,
-      minbucket = params$minbucket,
-      xval = params$xval,
-      maxdepth = params$maxdepth,
-      cp = params$cp
+convertDTParamsToArgs <- function(f_string, params) {
```

Comment:
That makes sense. 

On the topic of just keeping params as the params for `rpart`, it seems to me that for the sake of extendability, it's a lot more hoops to go through to pass any additional arguments (that might be needed for those other functions) separately.